colossalai/shardformer/layer/dropout.py

```diff
@@ -14,7 +14,7 @@ class SeedManager:
 
     def __init__(self):
         original_state = torch.cuda.get_rng_state()
-        seed = int(f"{int(time.time())}{os.environ['RANK']}")
+        seed = os.getpid()
```

Comment:
You can add a `TODO` note here to alert us that there will be some refactoring work to do in the future.

```python
# TODO: unify this seed manager with the colossalai.context.random
seed = os.getpid()
```